tests/unit/lib/test_opensearch_tls.py

```diff
@@ -235,6 +235,11 @@ def test_on_relation_created_non_admin(self, _, __, _request_certificate, deploy
             state=DeploymentState(value=State.ACTIVE),
         )
         event_mock = MagicMock()
+        self.secret_store.put_object(
```

Comment:
this was not directly added by me... But may have come when switching target branches...